completion/available/heroku.completion.bash

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+
+if command -v heroku > /dev/null; then
+    heroku autocomplete > /dev/null
```

Comment:
Why do you need both this line and then next? The Heroku documentation only mentions this line, not the next one...

Reply:
Its used to generate cache in `/home/$USER/.cache/heroku`

The next line is used to source those cache file